components/dashboard/src/app/AppBlockingFlows.tsx

```diff
@@ -53,5 +56,9 @@ export const AppBlockingFlows: FC = ({ children }) => {
         return <UserOnboarding user={user} />;
     }
 
+    if (!org.data) {
+        return <OrgNamingStep onComplete={() => {}} />;
```

Comment:
**issue(non-blocking):** Noticed we rely on the Dedicated org creation page, which is different from the org creation page (`/orgs/new`) and includes the two-step indication at the top. Maybe something that could be easily removed in this case.

| Org Creation on Dedicated |
|-|
| <img width="1512" alt="Frame 1524" src="https://github.com/gitpod-io/gitpod/assets/120486/921ec7d8-6417-4f4e-886f-eb6122099065"> |

Reply:
👍 There was a TODO for generalizing the progress indicator, which I did now. cc @selfcontained